src/TagsEditor.cpp

```diff
@@ -296,7 +296,7 @@ void TagsEditorDialog::PopulateOrExchange(ShuttleGui & S)
       S.EndHorizontalLay();
 
       if (mGrid == NULL) {
-         mGrid = safenew Grid(S.GetParent(),
+         mGrid = safenew Grid(FormatterContext::EmptyContext(), S.GetParent(),
```

Comment:
`TagsEditorDialog` is constructed only in `ShowEditDialog` and all calls to that have `AudacityProject&` near at hand, so likewise, they can have fuller context